genesis/src/address_generator.rs

```diff
@@ -3,28 +3,21 @@ use solana_sdk::{pubkey::Pubkey, system_instruction::create_address_with_seed};
 #[derive(Default)]
 pub struct AddressGenerator {
     base_pubkey: Pubkey,
-    base_seed: String,
     program_id: Pubkey,
     nth: usize,
 }
 
 impl AddressGenerator {
-    pub fn new(base_pubkey: &Pubkey, base_seed: &str, program_id: &Pubkey) -> Self {
+    pub fn new(base_pubkey: &Pubkey, program_id: &Pubkey) -> Self {
```

Comment:
Retaining support for base_seed seems nice, how about:
```suggestion
    pub fn new(base_pubkey: &Pubkey, base_seed: Option<&str>, program_id: &Pubkey) -> Self {
```

Reply:
Oh, nm.  I see we won't share this code with `solana create-address-with-seed` so whatever